### PR TITLE
feat(obi-1253): add aria labels and titles to necessary items

### PR DESCRIPTION
**Pull request by RunzelRosinchen**
## Proposed Changes

  - Add titles and aria-labels to necessary items

### CODE REVIEW

File: src/components/twoFactorAuth/TwoFactorAuth.tsx [hunk 1]
```diff
@@ -454,7 +454,8 @@ export const TwoFactorAuth = () => {
 					/>
 					<img
 						className="twoFactorAuth__qrCodeImage"
-						alt="qr code"
+						alt="QR code"
```
Comment (alxbenz) [inline src/components/twoFactorAuth/TwoFactorAuth.tsx]: maybe move this to the translation file aswell?

File: src/resources/i18n/de.ts [hunk 2]
```diff
@@ -1552,7 +1585,8 @@ export const de = {
 				headline: 'Ihr persönlicher QR-Code',
 				info: 'Wenn Sie Ihren QR-Code mit jemandem teilen, kann diese Person ihn mit der Handykamera scannen, um mit Ihnen direkt Kontakt aufzunehmen. Alternativ können Sie den Code auch herunterladen.'
 			}
-		}
+		},
+		'iconTitle': 'QR-Code'
```
Comment (alxbenz) [inline src/resources/i18n/de.ts:1589]: This could be used for the QR Code in my other comment.

File: src/resources/i18n/de.ts [hunk 3]
```diff
@@ -484,7 +495,8 @@ export const de = {
 					facts: '<ul><li>Was ist passiert?</li><li>Wie ist Ihre aktuelle Situation?</li><li>Was beschäftigt Sie?</li><li>Haben Sie eine bestimmte Frage oder wissen Sie vielleicht selbst noch nicht so genau was Ihnen helfen könnte?</li></ul>',
 					title: 'Vielleicht helfen Ihnen folgende Punkte bei der Formulierung weiter:'
 				},
-				headline: 'Hier ist Platz für Ihre Anliegen.'
+				headline: 'Hier ist Platz für Ihre Anliegen.',
```
Comment (alxbenz) [inline src/resources/i18n/de.ts:498]: Should also be translated for "de.informal".

File: src/resources/i18n/de.ts [hunk 4]
```diff
@@ -1269,7 +1297,11 @@ export const de = {
 	},
 	notifications: {
 		'message.new': 'Sie haben eine neue Nachricht!',
-		'enquiry.new': 'Sie haben eine neue Livechat Anfrage!'
+		'enquiry.new': 'Sie haben eine neue Livechat Anfrage!',
```
Comment (alxbenz) [inline src/resources/i18n/de.ts:1300]: Should also be translated for "de.informal".

**Review by alxbenz — CHANGES_REQUESTED** [review]

File: src/resources/i18n/de.ts (hunk 3 above)
Comment (RunzelRosinchen) [reply src/resources/i18n/de.ts:498]: I've just added the comma for the next line, which is the same for informal and formal, this one is already translated in de.informal

File: src/resources/i18n/de.ts (hunk 4 above)
Comment (RunzelRosinchen) [reply src/resources/i18n/de.ts:1300]: See comment above

**Review by alxbenz — APPROVED** [review]

**Pull request merged**